examples/2d/sprite_scale.rs

```diff
@@ -0,0 +1,139 @@
+//! Shows how to use sprite scaling modes to fill and fit textures into the sprite.
+
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_systems(Startup, setup)
+        .run();
+}
+
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
+    commands.spawn(Camera2d);
+    let font = asset_server.load("fonts/FiraSans-Bold.ttf");
+    let style = TextFont {
+        font: font.clone(),
+        ..default()
+    };
+
+    let square = asset_server.load("textures/slice_square_2.png");
+    let banner = asset_server.load("branding/banner.png");
+
+    let rects = vec![
```

Comment:
Ultra nit-picking, feel free to ignore xD 
- I don't think specifying the font is necessary any more, bevy has a default font built in
- Don't need a Vec here, it can just be an array `let rects = [ ... ];

Reply:
I'll go ahead and sort this out as soon as other's review comments appear. I just want to gather more comments for further work.